src/Sulu/Bundle/SecurityBundle/Resources/config/services.xml

```diff
@@ -45,11 +45,11 @@
 
             <tag name="sulu.context" context="admin"/>
         </service>
-        <service id="sulu_security.admin" class="Sulu\Bundle\SecurityBundle\Admin\SecurityAdmin">
+        <service id="sulu_security.admin" class="Sulu\Bundle\SecurityBundle\Admin\SecurityAdmin" lazy="true">
```

Comment:
This is an undiscovered circular dependency: 
* AdminPool loads all admins
* SecurityAdmin is one of them 
* SecurityAdmin requires the AdminPool